uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/WebTest/AbsencePeriod/FormTest.php

```diff
@@ -44,6 +44,7 @@ public function testCanAddPeriodWithMinimumRequiredFields() {
   public function testStartDateShouldBeLessThanEndDate() {
     $this->loginAsAdmin();
     $this->openAddForm();
+    $this->type('title', 'Test Absence Period');
```

Comment:
Is there any specific reason for this? It doesn't affect the test, since we're just checking the dates, I'm  just curious